package.json

```diff
@@ -20,8 +22,9 @@
   "scripts": {
     "prebuild": "rm -rf lib",
     "build": "tsc --outDir lib --project tsconfig-build.json",
+    "build:esm": "tsc --outDir esm --project tsconfig-esm.json",
     "lint": "tslint 'src/**/*.ts' --config tslint.json --project tsconfig.json && tsc --noEmit",
-    "prepare": "npm run build",
+    "prepare": "npm run build && npm run build:esm",
```

Comment:
Utilmately there should be 1 build task:
```json
{
    "build": "npm run build:cjs && npm run build:esm",
    "build:cjs": "tsc --outDir lib --project tsconfig-build.json",
    "build:esm": "tsc --outDir esm --project tsconfig-build.json --module esnext --target esnext",
    "prepare": "npm run build"
}
```

We should also just keep one `tsconfig` for the build, then just pass in the cli flags to adjust to what we need.

Reply:
updated :)